data/json/vehicleparts/motor.json

```diff
@@ -23,7 +23,7 @@
     "looks_like": "motor_small",
     "difficulty": 1,
     "durability": 80,
-    "power": 1492,
+    "power": 750,
     "energy_consumption": 1750,
```

Comment:
Why did this motor suddenly become half as efficient?

Reply:
My bad, forgot to adjust energy_consumption.